src/events/mod.rs

```diff
@@ -170,6 +170,23 @@ impl<'a> BytesStart<'a> {
         bytes.extend_from_slice(&*a.value);
         bytes.push(b'"');
     }
+
+    /// Edit the name of the BytesStart in-place
```

Comment:
Can you add a warning stating that the content of the `BytesStart` will be cleared?
Someone may not expect attributes to disappear.

Reply:
Good catch, I have added a warning.